values.yaml

```
@@ -1178,6 +1177,88 @@ ingressGateways:
   gateways:
     - name: ingress-gateway
 
+# Configuration options for terminating gateways. Default values for all
+# terminating gateways are defined in `terminatingGateways.defaults`. Any of
+# these values may be overridden in `terminatingGateways.gateways` for a
+# specific gateway with the exception of annotations. Annotations will
+# include both the default annotations and any additional ones defined
+# for a specific gateway.
+# Requirements: consul >= 1.8.0 and consul-k8s >= 0.16.0 if using
+# global.acls.manageSystemACLs.
+terminatingGateways:
+  # Enable terminating gateway deployment. Requires `connectInject.enabled=true`.
+  enabled: false
+
+  # Defaults sets default values for all gateway fields. With the exception
+  # of annotations, defining any of these values in the `gateways` list
+  # will override the default values provided here.
```

I thought it's worth repeating the same thing as you said in the comment for `terminatingGateways` here for clarity.
```suggestion
  # will override the default values provided here. Annotations will
  # include both the default annotations and any additional ones defined 
  # for a specific gateway.
```

---

Updated.